File: tmp_fix_tex.py
path = r"d:\repositorio_geral\app_gestao_contratos\DOCS\relatorio_institucional\capitulos\swebok.tex"

with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

# Let's count how many literal '\' characters there are or what combination
count_double = content.count('\\\\')
count_single = content.count('\\')

print(f"Count of '\\\\': {count_double}")
print(f"Total size: {len(content)}")

# Replace
fixed = content.replace('\\\\', '\\')

with open(path, 'w', encoding='utf-8') as f:
    f.write(fixed)

print("Replacement complete. Fixed successfully")

File: tmp_fix_tex.js
const fs = require('fs');
const path = "d:\\repositorio_geral\\app_gestao_contratos\\DOCS\\relatorio_institucional\\capitulos\\swebok.tex";

let content = fs.readFileSync(path, 'utf8');
// Replace double backslashes with support single backslash
let fixed = content.replace(/\\\\/g, '\\');
fs.writeFileSync(path, fixed, 'utf8');
console.log("Fixed successfully via Node");
